chromadb/db/index/hnswlib.py

```diff
@@ -244,10 +243,12 @@ def get_nearest_neighbors(
         # Check dimensionality
         self._check_dimensionality(query)
 
+        # Check Number of requested results
```

Comment:
This is the main change.

Reply:
we do not currently update this on `delete` :(

Reply:
Now we do!